bin/run-ansible.sh

```diff
@@ -71,6 +83,20 @@ echo "limit        = ${aurora_limit}"
 
 export ANSIBLE_ROLES_PATH="${aurora_home}/ansible/roles"
 
+extra_vars=$*
+IFS=',' read -ra inputdata <<< "$read_input"
+for i in "${inputdata[@]}"; do
+    printf "Username for $i:"
```

Comment:
message is misleading. It could something now related to username e.g. API key, AWS region etc